Automattic/wp-calypso
# Revert using component state for Image Editor Crop and fix underlying issue

Properly fixes #8286. Reverts #8296 and adds fix for underlying issue.

Initially, I thought this has something to do with using Redux for storing Image Editor Crop position/sizing as on each action dispatch, a lot of unrelated components to image editor are refreshed (even though their props were not updated).

However, I also noticed that this didn't happen some time ago (e.g. a month as said here: #8377). I went ahead and located the commit which introduced the regression. Turns out it was **I** who did this with [this line of code](https://github.com/Automattic/wp-calypso/pull/7913/commits/de1742e31fdcf215e64c4de1558f51e0e5ec2bf3#diff-d2ae10d6ec20ea5af5d82e33b56db5e6R66)! I apologize.

## Testing Instructions

1. Make sure the image editor works fine after applying this revert PR (cropping, rotating, changing aspect ratio, flipping, etc);
2. Follow the testing instructions in #8296;
3. Make sure that the issue described in #7988 is not back.

/cc @gwwar
